spec/dummy/app/themes/default/assets/stylesheets/thredded.scss

```diff
@@ -1,15 +1,25 @@
+@import url(http://fonts.googleapis.com/css?family=Lato:400,900,400italic);
+
 @import "bourbon";
-@import "base/base";
+@import "base/grid-settings";
 @import "neat";
-@import "thredded/variables";
-@import "thredded/common";
-@import "thredded/layout";
-@import "thredded/nav";
-@import "thredded/sidebar";
-@import "thredded/messageboards";
-@import "thredded/topics/common";
-@import "thredded/topics/public";
-@import "thredded/topics/private";
-@import "thredded/posts";
-@import "thredded/drop_forms";
+@import "base/base";
```

Comment:
So with regards to the top level directories/namespaces here -- base, utilities, components -- I had originally put them in the thredded directories to keep them isolated from any parent apps that *might* have a `base/_base.scss` file also. Do you think that would pose a problem if that was the case. I'm thinking it's an edge case we could mitigate (in the future, even). What do you think, though, about just shoving them one more dir in.

Or is that just an additional pain in the ass?